### DISCUSSION TITLE
Domain Renewal

### DISCUSSION THREAD

**Discussion by Cole-Krudwig** [vercel/community#7459] [Help]
### Summary

I began receiving emails that my billing info had expired and I needed to update it in order to renew my domain. I did this and updated my billing info to one that actually works. However, the expiration date came around and it did not renew for some reason. My billing info works and I would like to keep the domain.

### Steps to Reproduce

![Screenshot 2024-07-13 at 9 47 23 AM](https://github.com/user-attachments/assets/8a4c419f-f8f4-42c3-a568-19bf5528f5cf)

**Comment by amyegan** [comment]
Hey @Cole-Krudwig. Your domain is renewed.

I replied to your other post with more info - https://github.com/vercel/community/discussions/6604#discussioncomment-10040021

[a comment by an automated account is not shown]
